http-protocol-binding.md

```diff
@@ -429,6 +429,83 @@ Content-Length: nnnn
 
 ```
 
+### 3.4. Multipart Structured Content Mode
+
+In the _multipart structured_ content mode several events are sent into a single HTTP multipart
+request or response body and they can be named. A multipart structured mode contains events serialized 
+using one or several [event formats](#14-event-formats) in the same envelope. The _multipart structured_ content mode is based
+on [RFC7578 "Returning Values from Forms: multipart/form-data"](https://tools.ietf.org/html/rfc7578).
+
+#### 3.4.1. HTTP Content-Type
+
+The [HTTP `Content-Type`][content-type] header MUST be set to the media type of `multipart/form-data` 
+and must include the [boundary parameter](https://tools.ietf.org/html/rfc7578#section-4.1)
+
+```text
+Content-Type: multipart/form-data; boundary=12345
+```
+
+#### 3.4.2. Event Data Encoding
+
+Every part of the `multipart/form-data` can use its own [event format](#14-event-formats).
+
+The specific encoding of the part MUST be set using the `Content-Type` header inside the part:
+
+```text
+Content-Type: application/cloudevents+json; charset=UTF-8
+```
+
+#### 3.4.3. Naming the events
```

Comment:
Since `name` is optional in RFC7578, I am not sure whether name is _really_ needed here.
Could you please clarify the use of it?

Reply:
the reason for `name` is to allow to quickly identify an event without parsing it. An example use case is for invoking a "function service" with several events, and map the function parameters with the events in the envelope